### DISCUSSION TITLE
Multiple Number Walls on Lock screen?

### DISCUSSION THREAD

**Discussion by Foxy7** [NightscoutFoundation/xDrip#2046] [Q&A]
Hi,

is it possible to have two number walls showing on the lock screen?

I thought i'd be able to set up two number walls, using xDrip and xDrip variant, and they could show values at different heights on the screen, but it doesn't appear to be the case.

perhaps i could set the background to be transparent instead of black?

any ideas if this is possible somehow?

thanks

**Comment by Navid200** [comment]
@jamorham
